oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/VersionGCTest.java

```diff
@@ -531,6 +531,57 @@ public void testResetFullGCDryRunMode() throws Exception {
 
     // OAK-10370 END
 
+    // OAK-10896
+
+    @Test
+    public void testVersionGCLoadGCModeConfigurationNotApplicable() throws Exception {
+        int fullGcModeNotAllowedValue = 5;
+        int fullGcModeGapOrphans = 2;
+
+        // set fullGcMode to allowed value that is different than NONE
+        VersionGarbageCollector.setFullGcMode(fullGcModeGapOrphans);
+
+        // reinitialize VersionGarbageCollector with not allowed value
+        VersionGarbageCollector gc = new VersionGarbageCollector(
+                ns, new VersionGCSupport(store), true, false, false,
+                fullGcModeNotAllowedValue);
+
+        assertEquals("Starting VersionGarbageCollector with not applicable / not allowed value" +
+                "will set fullGcMode to default NONE", gc.getFullGcMode(), VersionGarbageCollector.FullGCMode.NONE);
```

Comment:
```suggestion
                "will set fullGcMode to default NONE", VersionGarbageCollector.getFullGcMode(), VersionGarbageCollector.FullGCMode.NONE);
```

static method should be accessed in a static way

Reply:
Fixed the comment, unit tests and static change, thank you for the suggestions